### DISCUSSION TITLE
How to display Jenkins last successful build

### DISCUSSION THREAD

**Discussion by josephjthomas** [badges/shields#7039] [Q&A] [answered]
How do you create a badge to show off the last successful build from Jenkins? I

**Comment by chris48s** [comment, marked as answer]
Hi. We've got 2 jenkins build badges which work with a jenkins job URL e.g: https://jenkins.sqlalchemy.org/job/alembic_gerrit

![](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.sqlalchemy.org%2Fjob%2Falembic_gerrit) - https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.sqlalchemy.org%2Fjob%2Falembic_gerrit
![](https://img.shields.io/jenkins/tests?compact_message&jobUrl=https%3A%2F%2Fjenkins.sqlalchemy.org%2Fjob%2Falembic_gerrit) - https://img.shields.io/jenkins/tests?compact_message&jobUrl=https%3A%2F%2Fjenkins.sqlalchemy.org%2Fjob%2Falembic_gerrit

**Reply by josephjthomas** [reply]
Hey,

thanks for the response. I am aware of that, but I was looking for a badge to display the version number of the last successful artifact, for example looking like "build: 1.0.0".

**Reply by calebcartwright** [reply]
@josephjthomas - you can always review our full badge offering by going to https://shields.io/ and using the search box in the center of the top of the badge (e.g. search for `jenkins` to see all jenkins badges) and/or clicking through each of the categories.

If a badge isn't listed there, then we don't have a badge for it. I'm going to mark Chris' answer as correct given the original question and because a badge that displays Jenkins Artifact Versions would be a net-new badge.

Such a request for a new badge would need  to be submitted via our standard new badge request route: https://github.com/badges/shields/issues/new?assignees=&labels=service-badge&template=3_Badge_request.md

If you'd like to see the addition of a new badge, please submit a corresponding request and be sure to provide the data points requested in the comments of the issue template.
